x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/edit_package_policy_page/index.tsx

```diff
@@ -288,16 +318,32 @@ export const EditPackagePolicyPage: React.FunctionComponent = () => {
             validationResults={validationResults!}
             submitAttempted={formState === 'INVALID'}
           />
+
+          {packagePolicy.policy_id &&
+            packagePolicy.package?.name &&
+            originalPackagePolicy &&
+            ExtensionView && (
+              <ExtensionWrapper>
+                <ExtensionView
+                  policy={originalPackagePolicy}
+                  newPolicy={packagePolicy}
+                  onChange={handleExtensionViewOnChange}
+                />
+              </ExtensionWrapper>
+            )}
```

Comment:
does it make more sense to add a conditional to either render `StepConfigurePackagePolicy` OR `ExtensionView` based on extension status? right now `StepConfigurePackagePolicy` [will return null](https://github.com/elastic/kibana/pull/82783/files?file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx&hide-deleted-files=true#diff-75c09954f4d914c04b7152c8568cd9a57a565912a4b3d207a07b24b3141ee792R128) only if there aren't valid policy templates AND and an extension is registered.

for endpoint the result is the same as there aren't inputs for the endpoint package, but other packages may have inputs but still want to override the form with a custom view. what do you think?

same comment goes for `create_package_policy_page/index.tsx`

Reply:
just realized this is basically the question posed in the other discussion, this is my take on it: https://github.com/elastic/kibana/issues/82482#issuecomment-725003803

Reply:
Thanks @jen-huang . Yeah, I have that other Issue to track the desired behaviour. Will comment on that one soon.